src/components/Directory/List.js

```diff
@@ -1422,4 +1422,11 @@ export const list = [
     description:
       "GoneBuster Geek T-shirts are made in Lyon, designed by a crazy team and produced with 100% biological fabric."
   },
+  {
+    name: "BitVoucher.cz",
+    url: "https://bitvoucher.cz/",
+    type: "service",
```

Comment:
```suggestion
    type: "services",
```
Missing a S in `services`
The comments at the very top have the right syntax and grammar for the entries to appear in the production UI, for reference.